infrastructure/dpladm/vpa-templates/cli-vpa.template.yaml

```diff
@@ -14,8 +14,8 @@ spec:
     containerPolicies:
     - containerName: cli
       minAllowed:
-        cpu: 4m
-        memory: 2Mi
+        cpu: 200m
+        memory: 200Mi
```

Comment:
Why do we need to raise the minimum?

We we discussed increased resource consumption I expect that raising the minimum resource allocation will have a much larger impact than raising the maximum allowed usage.

If this is related to vertical pod autoscaling then I would suggest that scaling is turned off for the CLI. If the VPA sets the maximum too low then it will effectively kill mid-run just as we are seeing now. As I see it it would be preferable that the memory limits are fixed.

Reply:
We did discuss just raising the limit and see if that is enough

Reply:
After reading up on VPA, I have a question: What's stopping the VPA from killing the pod to adjust limits mid-cron-run?

As I read it, the *minimum* ought to be raised to at least 400MiB, else we risk the VPA scaling the container down to below 400 due to inactivity, which then triggers k8n to kill the container before `drush` reaches its limit when encountering load.